### PR TITLE
Update admin table to have first and last name

### PR DESCRIPTION
**Pull request by michelle-vel**
Summary: 
- added first and last name to admin table
- added test
- added key to admin table map function
- fixed api url environment variable 

Screen Shot 2022-02-02 at 4 35 54 PM

### CODE REVIEW

**Review by onb6 — APPROVED** [review]
WOOT WOOT

**Comment by onb6** [comment]
> WOOT WOOT

This review probably shouldn't really count since we paired on this

File: src/components/researcherLandingPage/AdminTable.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const AdminTable: React.FC<AdminTableProps> = ({ data }) => (
       </Thead>
       <Tbody>
         {data.map((admin) => (
-          <AdminTableRow admin={admin} />
+          <AdminTableRow key={admin.email} admin={admin} />
```
Comment (rymaju) [inline src/components/researcherLandingPage/AdminTable.tsx:22]: 🔧 Can we change the table header from "Admin" to "Name"?

⭐ Bonus points if you make this table sortable by the name column as well

File: src/pages/researcherLandingPage/ResearcherLandingPage.test.tsx [hunk 2]
```diff
@@ -17,5 +17,9 @@ describe('Landing', () => {
     render(<ResearcherLandingPage />);
     const testEmail = await screen.findByText('test@test.com');
     expect(testEmail).toBeInTheDocument();
+    const testFirstName = await screen.findByText(/first/);
```
Comment (rymaju) [inline src/pages/researcherLandingPage/ResearcherLandingPage.test.tsx:20]: 💯

**Review by rymaju — CHANGES_REQUESTED** [review]
Very minor change, just want to also update the column header name. Otherwise great work!

File: src/components/researcherLandingPage/AdminTable.tsx (hunk 1 above)
Comment (onb6) [reply src/components/researcherLandingPage/AdminTable.tsx:22]: Fixed!

**Review by rymaju — APPROVED** [review]
LGTM

**Pull request merged**